src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -21,10 +21,15 @@ public class BuildEventArgsWriter
             typeof(LazyFormattedBuildEventArgs).GetField("arguments", BindingFlags.Instance | BindingFlags.NonPublic);
         private static FieldInfo buildEventArgsMessageField =
             typeof(BuildEventArgs).GetField("message", BindingFlags.Instance | BindingFlags.NonPublic);
-        private static Func<CultureInfo, string, object[], string> formatStringDelegate =
-            (Func<CultureInfo, string, object[], string>)Delegate.CreateDelegate(
-                typeof(Func<CultureInfo, string, object[], string>),
-                typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic));
+
+        // TODO: Delegate.CreateDelegate will only become available on .NET Standard 2.0, for now just use a MethodInfo
+        //private static Func<CultureInfo, string, object[], string> formatStringDelegate =
+        //    (Func<CultureInfo, string, object[], string>)Delegate.CreateDelegate(
+        //        typeof(Func<CultureInfo, string, object[], string>),
+        //        typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic));
+
+        private static MethodInfo formatStringMethodInfo =
+            typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic);
```

Comment:
You can use `MethodInfo.CreateDelegate` https://msdn.microsoft.com/en-us/library/hh194376(v=vs.110).aspx